src/encoder.rs

```diff
@@ -1146,7 +1146,19 @@ pub fn encode_block_b(fi: &FrameInvariants, fs: &mut FrameState,
             //let mode_context = if bo.x == 0 && bo.y == 0 { 0 } else if bo.x ==0 || bo.y == 0 { 51 } else { 85 };
             // NOTE: Until rav1e supports other inter modes than GLOBALMV
```

Comment:
Later, we remove this obsolete comment line.

Reply:
yes indeed